integration_tests/helpers/transactionBuilder.js

```diff
@@ -28,15 +28,82 @@ class TransactionBuilder {
     return Buffer.from(final).toString("hex");
   }
 
+  buildScriptChallenge(publicNonce, script, input_data, height) {
+    var KEY = null; // optional key
+    var OUTPUT_LENGTH = 32; // bytes
+    var context = blake2bInit(OUTPUT_LENGTH, KEY);
```

Comment:
nit: `var` [should never really be used in modern JS](https://javascript.info/var) - `const` is similar to `let` in rust (a little less restrictive, state can still change internal to the value), and `let` is similar to `let mut` in rust.
```suggestion
    const KEY = null; // optional key
    const OUTPUT_LENGTH = 32; // bytes
    const context = blake2bInit(OUTPUT_LENGTH, KEY); 
```